packages/core/src/generatePolicy.js

```diff
@@ -477,7 +487,8 @@ function aggregateDeps({ packageModules, moduleIdToModuleRecord }) {
           return
         }
         // moduleRecord missing, guess package name
-        const packageName = guessPackageName(requestedName)
+        const packageName =
```

Comment:
It's bothering me that I don't understand under what circumstances the `moduleRecord` would be missing.  My intuition is that it should not be tolerated, and whatever is consuming the `ModuleInspector` should ensure it's providing clean data.

It's also bothering me that a) we ultimately fallback to `<unknown:${requestedName}>` which may (must?) have some consequence when the app runs under Lavamoat, and b) I don't know what that consequence is. My best guess is "useless placeholder".  

That said, given we're generating policies here, I appreciate that there's some wiggle room.  

Also: if we're providing the user some sort of actionable feedback when we need to throw `<unknown:${requestedName}>` in the policy, I think I'd feel better about it.  If it's not actionable (and I don't see it referenced anywhere else in the repo) and serves no other purpose, why keep it?